### DISCUSSION TITLE
Why not use code-body in tiddlywiki.com

### DISCUSSION THREAD

**Discussion by kookma** [TiddlyWiki/TiddlyWiki5#6770] [Q&A] [answered]
In https://tiddlywiki.com I see the below in several tiddlers to show the code, specially for global macros (for example see $:/editions/tw5.com/style-guide-macros)

```
<pre><$view field="text"/></pre>
```

Is there any reason the `code-body` field with yes value is not used?

**Comment by pmario** [comment, marked as answer]
It wasn't available as we did add the PRE element. They should be changed now.

**Comment by kookma** [comment]
@pmario 
I think we can also overcome of ugly tiddler like below using one of hide-body or code-body

![image](https://user-images.githubusercontent.com/830394/178529354-b94e8b8e-9469-4700-9eb7-69494dc05288.png)

**Comment by Jermolene** [comment]
I think of setting the `code-body` field as a bit of blunt instrument. Where possible, I prefer extending the existing `$:/config/ViewTemplateBodyFilters/system` step in the view template body cascade.
